macros/models/_sector/dex/dex_atomic_arbitrages.sql

```diff
@@ -0,0 +1,129 @@
+{% macro dex_atomic_arbitrages(blockchain, transactions) %}
+
+-- Step 1: Check that the transaction contains at least 2 trades
+WITH multi_trade_txs AS (
+    SELECT block_time, tx_hash
+    FROM {{ ref('dex_trades') }}
+    WHERE blockchain = '{{blockchain}}'
+    {% if is_incremental() %}
+    AND block_time >= date_trunc('day', now() - interval '7' day)
```

Comment:
similar to last PR, let's update all incremental filters

Reply:
done